tests/reporting/formatting/test_resources/test_run_summary.md

```diff
@@ -1 +1 @@
-🧪 20 ❌ 2 💔 1 🙈 0
\ No newline at end of file
+🧪 20 ❌ 2 💔 1 🙈 0
```

Comment:
These changes probably break some unit tests since they are generated and then used to compare the output of functions, they already match precisely. Maybe exclude the test folder from the commit rules?

Reply:
Yeah, noticed that. I already had to exclude all the YAMLs, turns out Markdown files also have very finicky comparison rules.